src/Behat/Behat/Context/Snippet/Generator/ContextSnippetGenerator.php

```diff
@@ -122,7 +123,7 @@ public function generateSnippet(Environment $environment, StepNode $step): Snipp
      */
     private function getMethodName(string $contextClass, string $canonicalText, string $pattern): string
```

Comment:
The argument `canonicalText` should probably be renamed, as this is not what will be passed normally